src/components/Calculator.js

```diff
@@ -0,0 +1,43 @@
+import React, { PureComponent } from 'react';
+
+import Button from './Button';
+import './calculator.css';
+
+const signArray = [
+  'AC',
+  '+/-',
+  '%',
+  '/',
```

Comment:
- [ ] For the divide button,  Please use `÷` instead of `/` as on the design.

![image](https://user-images.githubusercontent.com/83803414/199511686-08b28a03-fee4-4973-9d97-790469720ec9.png)

Reply:
Hi @thecodechaser , Thanks for the feedback.